src/features/DisplayControls/Vitals/utils/VitalsUtils.js

```diff
@@ -1,15 +1,19 @@
 import axios from "axios";
 import React from "react";
 import { PATIENT_VITALS_URL } from "../../../../constants";
+import { defaultDateTimeFormat } from "../../../../constants";
 import { formatDate } from "../../../../utils/DateTimeUtils";
-import { formatDateAsString } from "../../../../utils/DateFormatter";
 import { FormattedMessage } from "react-intl";
 
 export const vitalsHeaders = {
   temperature: <FormattedMessage id={"Temperature"} defaultMessage={"Temp"} />,
-  bloodPressure: <FormattedMessage id={"BLOOD_PRESSURE"} defaultMessage={"BP"} />,
+  bloodPressure: (
```

Comment:
Any reason why we have to modify these intls? Can't we keep it like how we have for pulse and temperature?